src/frontend/app/AppState.ts

```diff
@@ -18,8 +18,8 @@ export enum SwitchState {
 }
 
 export interface SelectedIModel {
-  imodelName: string;
-  projectName: string;
+  imodelId: string;
+  projectId: string;

```

Comment:
Can we switch to project -> context everywhere as well?

Reply:
Done